app/views/topics/show.html.erb

```diff
@@ -19,7 +19,7 @@
                     community_topic_path(@community, @topic, anchor: "comments") %>
       </div>
       <br>
-      <p><%= @topic.description %></p>
+      <p><%= text_with_links(@topic.description).gsub("\n", '<br/>').html_safe %></p>
```

Comment:
I used the `text_with_links` helper method defined in `text_with_links_helper.rb`.

Reply:
gsub works fine but I'm really in favor of following the ruby style guide in this scenario about [not using `gsub` if we can find a more specialized alternative](https://github.com/bbatsov/ruby-style-guide#dont-abuse-gsub) like ActionView [`simple_format`](http://api.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html#method-i-simple_format)